### ISSUE TITLE
blockFromTag(...) fail with vine

### ISSUE THREAD

**Issue by HydrolienF** [kntrel/Underilla#3]
Hey, thanks for sharing this tools. I have manage to generate vanilla caves on an 1.19.4 spigot server following README steps with a really simple custom map.
I'm triing to generate vanilla caves with a map that I haven't create myself but it fail on oriented block as vines or pumkins.
it said that my vine `minecraft:vine[west=true,east=false,up=false,south=false,down=false,north=false]` does not have property 'down'. That's true, maybe  because my map have been create on an older minecraft version.

Is it possible to get an air block when createBlockData(...) fail instead of let the IllegalArgumentException crash the server?
I would love to send a pull request with this small fix, but I get compilation error when triing to build this repository with gradle: `package com.jkantrell.yamlizer.yaml does not exist`

Full stack trace:
```java.lang.IllegalArgumentException: Could not parse data: minecraft:vine[west=true,east=false,up=false,south=false,down=false,north=false]
        at org.bukkit.craftbukkit.v1_19_R3.block.data.CraftBlockData.newData(CraftBlockData.java:547)
        at org.bukkit.craftbukkit.v1_19_R3.CraftServer.createBlockData(CraftServer.java:2250)
        at org.bukkit.Bukkit.createBlockData(Bukkit.java:1853)
        at org.bukkit.Material.createBlockData(Material.java:4632)
        at com.jkantrell.mc.underilla.spigot.impl.BukkitChunkReader.blockFromTag(BukkitChunkReader.java:38)
        at com.jkantrell.mc.underilla.core.reader.ChunkReader.lambda$locationsOf$4(ChunkReader.java:148)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:178)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
        at com.jkantrell.mca.PaletteContainer.indexedEntriesOf(PaletteContainer.java:263)
        at com.jkantrell.mca.Section.getBlockLocations(Section.java:82)
        at com.jkantrell.mc.underilla.core.reader.ChunkReader.lambda$locationsOf$6(ChunkReader.java:147)
        at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:273)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
        at java.base/java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575)
        at java.base/java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260)
        at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616)
        at java.base/java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622)
        at java.base/java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627)
        at com.jkantrell.mc.underilla.core.reader.ChunkReader.locationsOf(ChunkReader.java:155)
        at com.jkantrell.mc.underilla.core.reader.ChunkReader.locationsOf(ChunkReader.java:125)
        at com.jkantrell.mc.underilla.core.generation.RelativeMerger.mergeLand(RelativeMerger.java:63)        at com.jkantrell.mc.underilla.core.generation.Generator.generateSurface(Generator.java:53)
        at com.jkantrell.mc.underilla.spigot.generation.UnderillaChunkGenerator.generateSurface(UnderillaChunkGenerator.java:57)
        at org.bukkit.craftbukkit.v1_19_R3.generator.CustomChunkGenerator.a(CustomChunkGenerator.java:146)
        at net.minecraft.world.level.chunk.ChunkStatus.lambda$8(ChunkStatus.java:126)
        at net.minecraft.world.level.chunk.ChunkStatus$d.doWork(ChunkStatus.java:334)
        at net.minecraft.world.level.chunk.ChunkStatus.a(ChunkStatus.java:276)
        at net.minecraft.server.level.PlayerChunkMap.lambda$31(PlayerChunkMap.java:723)
        at com.mojang.datafixers.util.Either$Left.map(Either.java:38)
        at net.minecraft.server.level.PlayerChunkMap.lambda$30(PlayerChunkMap.java:721)
        at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
        at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)        at net.minecraft.server.level.ChunkTaskQueueSorter.b(SourceFile:62)
        at net.minecraft.util.thread.ThreadedMailbox.h(SourceFile:91)
        at net.minecraft.util.thread.ThreadedMailbox.a(SourceFile:146)
        at net.minecraft.util.thread.ThreadedMailbox.run(SourceFile:102)
        at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: com.mojang.brigadier.exceptions.CommandSyntaxException: Block minecraft:vine does not have property 'down' at position 57: ...uth=false,<--[HERE]```

**Comment by Jeshuakrc** [comment]
Hey!
The build issue had to do with [this](https://github.com/Jeshuakrc/Yamlizer) repo. I had been developing using the dev branch localy cached all along, and didn't realized I hadn't event merged to main LoL.
Yamelizer's main is up to date now, you should be able to build localy by deleting your gradle caches and runing the jar task.

I'm on the actual fix for `BukkitChunkReader#blockFromTag` now.

**Comment by HydrolienF** [comment]
Thanks for your answer ! I now can build.

I have find a way to make it work with current version: Prerender everychunck without Underilla. Then use prerender world to rerender it with Underilla.
That way all blocs have up to date properties before being rendered by Underilla.

If you find a way to ignore missing properties it will be great !

**Comment by Jeshuakrc** [comment]
Fixed in [v1.1.1](https://github.com/Jeshuakrc/Underilla/releases/tag/1.1.1)
The Spigot implemenation uses the `org.bukkit.Material#createBlockData` method to deserialize blockData strings to blocks, so I have no control over how missing properties are handled.
Instead, the `IlegalArgumentException` is catched, and a block with the same material is placed but with no actual data. So intead of just placing an Air block, you'll get a vine with defauld oriantation.

Thanks for bringing this up!

**Comment by HydrolienF** [comment]
Thanks a lot !
